public/avatar-generator/main.js

```diff
@@ -0,0 +1,116 @@
+// Run the code as soon as the window loads
+window.onload = function () {
+  // Add section of code for each body part to generate a random image file name
+
+  // NECK
+  const neck = new Image()
```

Comment:
to avoid linter errors w/these new Image declarations u can either add `/* global Image */` to the top of the file, or u can be explicit about where this Image class is located (ie. the global window object) && write it like this:
```js
const neck = new window.Image()
```

Reply:
added and pushed!